funcx_endpoint/funcx_endpoint/endpoint/interchange.py

```diff
@@ -210,14 +210,15 @@ def migrate_tasks_to_internal(
                 f" as {endpoint_uuid}"
             )
             task_q_proc = TaskQueueSubscriber(
-                pika_conn_params=connection_params,
+                conn_params=connection_params,
                 external_queue=pending_task_queue,
                 kill_event=quiesce_event,
-                endpoint_uuid=endpoint_uuid,
+                endpoint_id=endpoint_uuid,
             )
             task_q_proc.start()
         except Exception:
             log.exception("Unhandled exception in TaskQueueSubscriber")
+            raise
```

Comment:
I inserted this because the above was failing and then hiding the actual error when I ran into issues in my runs.

If the `except` clause is running, then `return task_q_proc` below will NameError.

I think it would be better not to catch this at all, but... not right now.